scania-digital-design-system/sdds
# feat(block): Migrate block

**Describe pull-request**  
fix: migrated block to new storybook

**Solving issue**  
Fixes: [AB#2392](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2392)

**How to test**  
1. Check the component in the new storybook
2. Check if its migrated in accordance with the migrate-checklist
